docker/web/build.gradle

```diff
@@ -9,7 +9,7 @@ configurations {
     js
 }
 dependencies {
-    js project(path: ':web', targetConfiguration: 'js')
+    js project(path: ':web-js', targetConfiguration: 'js')
```

Comment:
Assuming this project does NOT produce java artifacts, this is fine.  If this does in any way get published w/ jars, then we can't use `targetConfiguration` and expect transitivity to work.